lib/scsi_defs.h

```diff
@@ -80,5 +97,282 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 95 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct Read6 {
```

Comment:
Nit: prefer to stay with a consistent style when defining command structs. In other PRs we have stuck to the standard of having all command structs suffixed with "Command" i.e. `Read6Command`